inspire-atom/src/main/java/org/fao/geonet/inspireatom/util/InspireAtomUtil.java

```diff
@@ -30,6 +30,7 @@
 import java.util.List;
 import java.util.Map;
 
+import javassist.NotFoundException;
```

Comment:
Just notice this, is this import correct?

Reply:
https://github.com/geonetwork/core-geonetwork/pull/4228